_content/av/food-of-sinful-demons_barstow-geoff.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Food of Sinful Demons"
+authors:
+  - "Geoff Barstow"
+external_url: "https://podcasts.apple.com/us/podcast/new-books-in-buddhist-studies/id458210899?i=1000451889741"
+source_url: "https://newbooksnetwork.com/geoffrey-barstow-food-of-sinful-demons-meat-vegetarianism-and-the-limits-of-buddhism-in-tibet-columbia-up-2018/"
```

Comment:
Let's make the NBN link the external_url and not use the proprietary Apple link.